Makefile

```diff
@@ -0,0 +1,20 @@
+MODULE = gammapy
+
+CURDIR = $(shell pwd)
```

Comment:
I think this can be removed and the Makefile will just work the same?
(see the one in ctapipe)